pkg/services/ssosettings/ssosettings.go

```diff
@@ -9,10 +9,6 @@ import (
 )
 
 var (
-	// ConfigurableOAuthProviders is a list of OAuth providers that can be configured from the API
-	// TODO: make it configurable
-	ConfigurableOAuthProviders = []string{"github", "gitlab", "google", "generic_oauth", "azuread", "okta"}
-
```

Comment:
This code is no longer needed or used anywhere. The list of configurable IdPs can be set at `[sso_settings][configurable_providers]`